VMs/metrics/README.md

```diff
@@ -33,6 +33,22 @@ Briefly, the scripts do the following:
 - Provide scripts for use from Jenkins to launch a cloned VM with a Jenkins agent and to
 delete a cloned VM.
 
+## Pre-requisites
+
+The scripts in this directory rely upon certain tools being available on the host
+system. The following lists those tools, and the Ubuntu packages or alternative
+installation methods that can be used to install them. Package names may be
+different for other distributions.
+
+| Dependency | Ubuntu package or method |
+| ----------- | ------------------------ |
+| go | Install latest from [golang.org](golang.org) |
+| yq | Install latest with `go get` from [mikefarah/yq](github.com/mikefarah/yq)|
+| qemu-kvm | package qemu-kvm |
+| libvirt | package libvirt-bin |
+| virtinst | package virtinst |
+| jre | package default-jre |
```

Comment:
OOI, does that equate to the openjre or does it need to be another implementation (Oracle?)

Reply:
afaict from a quick look on a local machine, i believe it is openjdk. We just need 'some java' to launch the jenkins agent.jar with. the default-jre on ubuntu wfm so far :-)